package.json

```diff
@@ -1,15 +1,15 @@
 {
-    "name": "timelord",
-    "version": "0.1.0",
+    "name": "time-lord-bot",
+    "version": "0.0.1",
+    "description": "A discord bot",
+    "main": "index.js",
+    "author": "Errolyn",
+    "license": "MIT",
     "scripts": {
-        "start": "node bot.js"
+      "start": "node index.js"
```

Comment:
I *think*, but I'm not sure that if you set `"main": "index.js"`, and remove this line, then `yarn start` (and `npm start`) work automatically.

Reply:
That did not work for me, but I could be missing something?